blazar/plugins/devices/device_plugin.py

```diff
@@ -53,6 +53,14 @@
                default='',
                help='Default resource_properties when creating a lease of '
                     'this type.'),
+    cfg.BoolOpt('display_default_resource_properties',
+                default=False,
```

Comment:
It seems to me that both of these options are better to default for True, as they create the best end-user experience. What do you think? Given that they are also conditional on having set default resource properties at all, it won't change any default behavior for existing deployments.

Reply:
I agree, default True makes sense. It won't reveal any behavior unless the default_resource_properties is set.